src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -215,7 +255,9 @@ class ResourceService(
   private def constructAccessPolicy(resourceType: ResourceType, resourceId: ResourceId, validatableAccessPolicy: ValidatableAccessPolicy, public: Boolean) =
     AccessPolicy(
       FullyQualifiedPolicyId(FullyQualifiedResourceId(resourceType.name, resourceId), validatableAccessPolicy.policyName),
-      validatableAccessPolicy.emailsToSubjects.values.flatten.toSet,
+      validatableAccessPolicy.emailsToSubjects.values.flatten.toSet ++ validatableAccessPolicy.memberPolicies
+        .getOrElse(Set.empty)
+        .map(_.toFullyQualifiedPolicyId),
```

Comment:
See [comment below](https://github.com/broadinstitute/sam/pull/1613/files#r1894413066). `memberPolicies` are no longer included in `emailsToSubjects` so add them here when constructing the policy